roll/__init__.py

```diff
@@ -399,9 +533,11 @@ class Roll:
     Response = Response
     Cookies = Cookies
 
+
     def __init__(self):
         self.routes = self.Routes()
-        self.hooks = {}
+        self.hooks = dict()
```

Comment:
Warning: `dict()` is slower than literal `{}`.
On Roll init I think it's fair tough. :)